processor/src/main/java/org/mapstruct/ap/internal/model/PropertyMapping.java

```diff
@@ -806,7 +781,7 @@ public JavaExpressionMappingBuilder javaExpression(String javaExpression) {
         }
 
         public PropertyMapping build() {
-            Assignment assignment = new SourceRHS( javaExpression, null, existingVariableNames );
+            Assignment assignment = new SourceRHS( javaExpression, null, existingVariableNames, "" );
```

Comment:
its what it says it is. Its used to identify the source part of an error message. I could use a better name (struggled with it). Ideas?